distributed/shuffle/tests/test_shuffle.py

```diff
@@ -14,20 +14,20 @@
 import pytest
 
 pd = pytest.importorskip("pandas")
+dd = pytest.importorskip("dask.dataframe")
```

Comment:
This should be equivalent ot the pandas importorskip

Reply:
I've seen the native+dask version and the native-version only version of this skipping and wasn't sure which one we want.